### PR TITLE
[CDS-487][Defect][Accessibility] Accordion - User should use tab key to navigate instead of arrow keys

### PR DESCRIPTION
**Pull request by yagin94**
# Merge request description template

## Description

User should use tab key to navigate instead of arrow keys follow w3

**1) Changes**

https://comfortdelgrotaxi.atlassian.net/browse/CDS-487

**2) Impact**

## Check list

- [x] 1. Did you start and verify your changes?
- [x] 2. Did you run build to make sure that your changes can be built successfully?
- [x] 3. No !important flag, inline style in css
- [x] 4. No bolerplate codes (1)
- [x] 5. No duplicate code that exist in common functions?
- [x] 6. All of defined variables should have default value, check null and undefined?
- [x] 7. Use meaningful name for variables, no suffix 1,2,... Describes reference information for easier to understand what's inside. (2)
- [x] 9. Unsubscribed all of subscribers and even listeners when you don't need them.

### CODE REVIEW

File: packages/react-compass/src/accordion/accordion-button.tsx [hunk 1]
```diff
@@ -33,6 +33,8 @@ const AccordionButton = React.forwardRef<HTMLButtonElement, ButtonProps>(
 
     return (
       <StyledAccordionButton
+        aria-expanded={expand === 'close' ? false : true}
```
Comment (fpt-duongtx) [inline packages/react-compass/src/accordion/accordion-button.tsx:36]: aria-expanded={expand !== 'close'}

Comment (fpt-duongtx) [inline packages/react-compass/src/accordion/accordion-button.tsx:36]: Use camel case please!

File: packages/react-compass/src/accordion/accordion-title.tsx [hunk 2]
```diff
@@ -47,20 +50,14 @@ const AccordionTitle = React.forwardRef<HTMLButtonElement, AccordionTitleProps>(
 
     const handleKeyDown = (e?: unknown) => {
       const event = e as React.KeyboardEvent<HTMLElement>
-      const {key} = event
-      switch (key) {
-        case 'Spacebar':
+      const {code} = event
+      switch (code) {
+        case 'Space':
```
Comment (fpt-duongtx) [inline packages/react-compass/src/accordion/accordion-title.tsx:55]: Move event key out to constants.ts, KeyBoard. vairable name should not be code, it should be key

**Review by fpt-duongtx — DISMISSED** [review]

File: packages/react-compass/src/accordion/accordion-button.tsx (hunk 1 above)
Comment (yagin94) [reply packages/react-compass/src/accordion/accordion-button.tsx:36]: React type definitions are case-sensitive for ARIA attributes

**Review by Tai-Chi-Bui — APPROVED** [review]

**Pull request merged**